root/etc/s6-overlay/s6-rc.d/init-custom-files/run

```diff
@@ -36,10 +30,14 @@ if [[ -e "${SCRIPTS_DIR}" ]] && [[ -n "$(/bin/ls -A ${SCRIPTS_DIR} 2>/dev/null)"
             echo "[custom-init] ${NAME}: is not a file"
         fi
     done
-fi
 
-if [[ -e "${SCRIPTS_DIR_OLD}" ]] && [[ -n "$(/bin/ls -A ${SCRIPTS_DIR_OLD} 2>/dev/null)" ]]; then
-    echo "[custom-init] files found, executing"
+    # Remove legacy folder if it's empty
+    if [[ -e "${SCRIPTS_DIR}" ]] && [[ -z "$(/bin/ls -A ${SCRIPTS_DIR} 2>/dev/null)" ]]; then
+        echo "[custom-init] Legacy service folder ${SCRIPTS_DIR} is empty, deleting..."
+        rm -rf "${SCRIPTS_DIR}"
+    fi 
```

Comment:
Are these meant to be `SCRIPTS_DIR_OLD`? Or is the comment (wording: "legacy") incorrect here?